File: ui/src/store/modules/message.js
// 会话消息  
const message = {
  state: { 
    type:'',
    data:{
        sessionId:null,//会话id
        flagId:null,//假id
        isFavorite:0,
        text:'',
        realId:null,//消息id
        time:null,//消息时间
    }
  },
  mutations: { 
    SET_TYPE(state,type){
        state.type = type
    },
    SET_DATA(state,data){
        state.data = data
    }
  },
  actions: { 
    
  }
}
 
export default message

File: RuoYi-Vue-master/ruoyi-chat/src/main/java/com/chat/ruoyichat/domain/dto/Message.java
package com.chat.ruoyichat.domain.dto;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;
import lombok.ToString;

import java.util.Date;

/**
 * @author chat
 * @since 2025/3/17
 **/
@Data
@AllArgsConstructor
@NoArgsConstructor
@ToString
public class Message {
    private String sessionId;
    private String text;
    private Date time;
    private Long success;//1成功 0失败
    private String flagId;
    private Long realId;
    private Long isFavorite;
}
